app/classifier/tasks/survey/choice.cjsx

```diff
@@ -52,7 +52,7 @@ module.exports = React.createClass
     onCancel: Function.prototype
 
   getInitialState: ->
-    answers: {}
+    answers: @props.annotation?.answers || {}
```

Comment:
I think a `propType` definition would be helpful for anyone who has to maintain this code. I'm not fussed about a default value, but eslint nags you to add one when a prop isn't required, so that's probably a good practice to get into.